pkg/kubelet/kubelet.go

```diff
@@ -1590,23 +1620,27 @@ func getPodReadyCondition(spec *api.PodSpec, info api.PodInfo) []api.PodConditio
 	return ready
 }
 
-// GetPodStatus returns information from Docker about the containers in a pod
-func (kl *Kubelet) GetPodStatus(podFullName string, uid types.UID) (api.PodStatus, error) {
-	var spec api.PodSpec
-	var podStatus api.PodStatus
-	found := false
+func (kl *Kubelet) GetPodByFullName(podFullName string) (*api.PodSpec, bool) {
```

Comment:
For consistency I believe it should be nere GetPodByName, and return *?api.BoundPod.